content/blog/2018/05/2018-05-17-tracymiranda-intro.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Introducing Tracy Miranda as the CloudBees Open Source Program Lead"
+tags:
+- community
+author: tracymiranda
+---
+
+I’m Tracy Miranda, and I’m really excited to have joined CloudBees this month leading the open source program. CloudBees’ contributions to Jenkins include developing Pipeline and Blue Ocean, staffing the infrastructure team, advocacy and events work, as well as security efforts. My focus is on making sure there is a great relationship between the Jenkins community and CloudBees, which means strong communication, help get traction on things the community wants, and generally working to make Jenkins and the community thrive and stay awesome in an ever-changing tech landscape. 
+
+Here’s a little background on me: I come from an electronics/EDA background but switched to software early in my career when I first got involved with open source software. I’ve been part of the Eclipse community for around 15 years, definitely from before git was even a thing. I love being involved with all levels: project committer, conference chair, steering committee for working groups and more recently board of directors.
+
+On a personal note, I …
+
+* Live in the UK with my husband and 2 young kids
+* Grew up in Kenya
+* Enjoy playing badminton, love good food & am always first at any buffets
+
+I am looking forward to getting to know the Jenkins community well, and really getting a feel for your Jenkins stories, good and bad. Please feel free to let me know:
+
+* What you love about the Jenkins community & how you are using Jenkins
+* What you’re working on doing with Jenkins
+* What you don’t like and want improved
+
+You can find me on the mailing lists or via:
```

Comment:
Link to the mailing lists?
https://jenkins.io/mailing-lists/

Reply:
Would be nice to also suggest a specific ML for such conversations. We do not have a "community" mailing list so far, but maybe DEV list or Events could work

Reply:
Good tip - I'll add in link to mailing list. Yes I'd prefer to use the existing lists rather than have folks come to a new list.